### ISSUE TITLE
TF TinyMacros erwähnen 

### ISSUE THREAD

**Issue by Kebap** [MorgenGrauen/mg-web#31]
## 2019-07-03 

Es geht um die [Clientseite zu TinyFudge](http://mg.mud.de/download/tf.shtml)
* Bislang sind Ahab/Taranis verlinkt
  * bleiben auch so bestehen
  * aber die haben nur wenige Macros.
* TinyMacros fehlen dort komplett
  * [x] Plan ist sie dort ebenfalls zu erwähnen
  * Die sind überhaupt nicht mehr tiny sondern stark gewachsen
  * Croft sagt, das Github Repo von Mesirii hat eine gute Einführung, ist aber veraltet. Chagall hat eine aktuellere Version in seinem Profil verlinkt.
    * Mesi: https://github.com/jexp/TinyMacros
    * Chagall: https://github.com/chrib/TinyMacros
* [x] Befrage Chagall, ob Verlinkung genehm wäre. 
  * Chagall ist einverstanden: Das Github-Repo kann man gerne verlinken.
* [x] Bitte Croft um kurze Beschreibung
  * dahinter verbergen sich Mapper, Wegesystem und hastdunichtgesehen
  * /untroom, �vorsicht-handling mit macroflucht   
* Nicht sicher, ob ich /untroom bewerben möchte...

**Comment by Kebap** [comment]
--> MIt @jexp und @chrib klären, wessen Seite bestes Linkziel wäre

**Comment by jexp** [comment]
Von mir aus gern meins :) hab die letzten PRs gemerged, wäre auch eigentlich sinnvoll die wieder zusammenzuführen. Ich kann ja mal sehen welche commits da bei Chagall noch drin sind die bei mir fehlen. Mesi

**Comment by chrib** [comment]
Wenn es nach mir geht dann meins. Aber Spass beiseiten, das meiste im Repo ist eh von @jexp (Dotz und Thufhir nicht vergessen), deswegen kann man ihn ruhig verlinken. Am besten noch schauen was im ursprünglichen Repo noch eventuell fehlt und noch mit aufnehmen, falls es noch nicht geschehen ist.

**Comment by Kebap** [comment]
> Von mir aus gern meins :)

> Wenn es nach mir geht dann meins. 

Ihr seid ja lustig. 😄 

Wenn ihr euch auf einen Ort einigt, kann man ein Repo ja auch mit mehreren Leuten verwalten.

Am wichtigsten scheint mir, dass auch in 2-3 Monaten oder Jahren noch Fixes entwickelt oder angenommen werden.

Ansonsten haben wir dieses Gespräch dann schon wieder und müssen ggf. den Link ändern.

Aber gut, dann stelle ich jetzt erstmal den Link auf @jexp auf die Webseite.

> Am besten noch schauen was im ursprünglichen Repo noch eventuell fehlt und noch mit aufnehmen, falls es noch nicht geschehen ist.

Machst du das? Kann Github das nicht sogar vielleicht automatisch unterstützen?

**Comment by jexp** [comment]
Ja, @chrib kann die deltas von dem Fork als pull-request schicken.

**Comment by Kebap** [comment]
http://mg.mud.de/download/tf.shtml erwähnt jetzt die TinyMacros.
Bitte prüft (und ergänzt ggf.) meine stümperhafte Beschreibung dort...

**Comment by chrib** [comment]
> Ja, @chrib kann die deltas von dem Fork als pull-request schicken.

Komme wahrscheinlich erst die nächsten Tage dazu, aber ich probier mal mein Glück mit nem Pull-Request
